app/Livewire/Dashboard.php

```
@@ -16,15 +19,85 @@ class Dashboard extends Component
+        if ($userId === null) {
+            return [];
+        }
+
+        return [
+            "echo-private:user.{$userId},.RecentsUpdated" => 'refreshPinnedPages',
+        ];
+    }
+
     public function mount()
     {
         $this->privateKeys = PrivateKey::ownedByCurrentTeamCached();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\PrivateKey' in method 'mount'. (undefined)

(StaticAccess)